tests/unit/test_charm.py

```diff
@@ -515,36 +516,33 @@ def test_delete_password(self):
                 in self._caplog.text
             )
 
-    @patch("charm.JujuVersion.has_secrets", new_callable=PropertyMock, return_value=True)
     @pytest.mark.usefixtures("use_caplog")
+    @patch("charm.JujuVersion.has_secrets", new_callable=PropertyMock, return_value=True)
     def test_delete_existing_password_secrets(self, _):
         """NOTE: currently ops.testing seems to allow for non-leader to remove secrets too!"""
         with self.harness.hooks_disabled():
             self.harness.set_leader(True)
-        self.harness.charm.set_secret("app", "operator-password", "somepw")
-        self.harness.charm.remove_secret("app", "operator-password")
-        assert self.harness.charm.get_secret("app", "operator-password") is None
+        self.harness.charm.set_secret("app", "monitoring-password", "somepw")
```

Comment:
Quick question: is `monitoring-password` indeed the thing that exists in the charm?

(Sry I forgot which charm has which keywords :sweat_smile: )

Reply:
yep!

Reply:
Cool, thx :-)